instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientAttributesExtractorTest.java

```diff
@@ -145,14 +145,33 @@ public void stripBasicAuthTest() {
     Map<String, String> request = new HashMap<>();
     request.put("url", "https://user1:secret@github.com");
 
+    stripRequestTest(request, "https://github.com");
+  }
+
+  @Test
+  public void stripBasicAuthWithQueryParamTest() {
```

Comment:
JUnit 5 test methods need not be public
```suggestion
  void stripBasicAuthWithQueryParamTest() {
```

Reply:
Fixed that